.gitignore

```diff
@@ -69,3 +69,4 @@ kratos/includes/kratos_version.h
 *.bak
 
 applications/python_scripts/beam_profiles\.csv
+applications/DelaunayMeshingApplication/external_libraries/
```

Comment:
this should go to your local gitignore

Same as here: https://github.com/KratosMultiphysics/Kratos/tree/master/applications/ShapeOptimizationApplication/test_examples

Reply:
Didn't know you could have local ignores. In this case yep, that's better

Reply:
ok